lib/providers/buildings_data.dart

```diff
@@ -1,16 +1,13 @@
-import 'dart:typed_data';
-
 import 'package:concordia_navigation/models/outdoor/building.dart';
-import 'package:concordia_navigation/services/search.dart';
-import 'package:flutter/services.dart';
+import 'package:concordia_navigation/models/university.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
-import 'dart:ui' as ui;
 
 ///Observer Pattern
 ///Handles data related to campus buildings, listens to changes and notifies listeners.
 class BuildingsData extends ChangeNotifier {
-  static List<Building> allBuildings = [];
+  List<Building> allBuildings = [];
```

Comment:
This is going to change how we load floors for indoor navigation. I'll check if this change doesn't break anything that can't be fixed.